doc/docs/master/concepts/pipeline-concepts/datum/index.md

```diff
@@ -5,63 +5,77 @@
     all data at once, each directory independently, individual
     files one by one, or combined data from multiple inputs together.
 
-A datum is the smallest indivisible unit of computation within a
-job.
+## Definition
+A datum is **the smallest indivisible unit of computation within a job**.
 A job can have one, many or no datums. Each datum is processed
-independently with a single execution of the user code and
-then the results of all the datums are merged together to
+independently with a single execution of the user code on one of
+the pipeline worker pods.
+The files output by all of the datums are then combined together to
 create the final output commit.
 
-TODO: Inputs are based on branches, not repositories. Although that may
-be a common way to think about it since people will typically just use the master branch as input,
-it is an important distinction that should be clear in the docs.  
+## Data distribution
+Think of datums as a way to **divide your input data** 
+and **distribute processing workloads**.
+They are instrumental in optimizing your pipeline performance.
 
-A datum defines the input data. An input can take one or multiple
-repositories. Pachyderm has the following types of inputs that
-combine multiple repositories:
+You define how your data is spread among workers by
+**specifying [pipeline inputs](#pipeline-inputs)** for your pipeline 
+in its pipeline specification file. 
 
-TODO: I would focus more on these inputs being a composition of other inputs rather than
-referring to branches being combined in a certain way. It seems like the PFS input should be mentioned first,
-then the below inputs could be described as a way of composing inputs with a PFS input usually being the most
-primitive input. 
+Based on this specification file, the data in the `input` 
```

Comment:
We should remove this or replace it with just saying the input describes how to turn the input data into datums. Describing the process is leaking details that are not generally correct (the globbing only applies to PFS inputs). Also, I would not consider globbing in a PFS input as breaking down the data.